pyodata/v2/model.py

```diff
@@ -2159,9 +2160,3 @@ def update_global_variables_with_alias(aliases):
             namespace, suffix = direction_key.rsplit('.', 1)
             for alias in aliases[namespace]:
                 SAP_VALUE_HELPER_DIRECTIONS[alias + '.' + suffix] = SAP_VALUE_HELPER_DIRECTIONS[direction_key]
-
-
-def schema_from_xml(metadata_xml):
```

Comment:
This is a facade method and I would like to ask you to add it back because the goal was to hide the implementation details like we use the class `Edmx`.

Reply:
ok. reverted